.github/workflows/main.yml

```diff
@@ -1,4 +1,7 @@
-on: workflow_dispatch
+on:   
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule 
+  schedule:
+    - cron: "0 6 * * 1"
 
 jobs:
   trigger-script-test:
```

Comment:
> run: curl -X POST -d "{\"request\":{\"branch\":\"**master**\ 


ℹ️ : Was this meant to be run on master, or is it temporary ?

Reply:
Yes as it is easier that way, instead of having two branches to maintain